charts/redpanda/values_partial_test.go

```diff
@@ -53,6 +54,11 @@ func TestPartialValuesRoundTrip(t *testing.T) {
 			unstructuredJSON, err := json.Marshal(unstructuredValues)
 			require.NoError(t, err)
 
+			t.Logf("structured: %s", string(structuredJSON))
+			fmt.Printf("structured: %s\n", string(structuredJSON))
```

Comment:
Left over `fmt.Printf`s? The `t.Log`s should be sufficient, no?

Reply:
oh no, i meant to remove that!